extract_assets.py

```diff
@@ -68,25 +69,47 @@ def ExtractFunc(fullPath):
             globalExtractedAssetsTracker[fullPath] = globalManager.dict()
         globalExtractedAssetsTracker[fullPath]["timestamp"] = currentTimeStamp
 
-def initializeWorker(abort, unaccounted: bool, extractedAssetsTracker: dict, manager):
+def initializeWorker(abort, unaccounted: bool, extractedAssetsTracker: dict, manager, baseromSegmentsDir: Path, outputDir: Path):
     global globalAbort
     global globalUnaccounted
     global globalExtractedAssetsTracker
     global globalManager
+    global globalBaseromSegmentsDir
+    global globalOutputDir
```

Comment:
Using globals for passing flags :despair:

I guess its fine since the script was already doing this

Reply:
this script is on a suspended yeet sentence in OoT (see new assets system tm) so I really don't care about it yeah, I don't want to clean it up completely (would be basically a rewrite)